src/components/posts/PostsPage.js

```diff
@@ -33,20 +35,31 @@ const PostsPage = ({role}) => {
     });
   };
 
+  const handleClick = (user) => {
+    const id = user.id;
+    navigate(`/user/${id}`);
+  };
+
   return (
     <>
-      <Header role={role}/>
+      <Header role={role} />
       <section className='posts-section'>
         <p>Status: {postResponse.status}</p>
         <PostForm handleSubmit={createPost} handleChange={handleChange} />
-        <ul className="posts-list">
+        <ul className='posts-list'>
           {posts.map((post, index) => (
-            <li key={index} className="post-item">
+            <li key={index} className='post-item'>
               <Box>
-                <div className="post-content">{post.content}</div>
-                <Stack spacing={2} direction="row">
-                  <Box variant="contained">{`${post.user.profile.firstName} ${post.user.profile.lastName}`}</Box>
-                  <Box variant="contained">
+                <div className='post-content'>{post.content}</div>
+                <Stack spacing={2} direction='row'>
+                  <Box
+                    className='post-author'
+                    variant='contained'
+                    onClick={() => handleClick(post.user)}
+                  >
+                    <strong>{`${post.user.profile.firstName} ${post.user.profile.lastName}`}</strong>
```

Comment:
Rather than using an onClick handler and calling the naviate method, is it possible to wrap the contents in a react router <Link /> element? The benefits to this would be less code, and also the generated HTML will actually have an <a> element so it's a bit more semantic. If it's a difficult change, not a bit deal.

Reply:
Great suggestion, this is far easier.